google/resource_container_cluster.go

```diff
@@ -266,6 +271,60 @@ func resourceContainerCluster() *schema.Resource {
 						},
 
 						"node_config": schemaNodeConfig,
+
+						"management": {
+							Type:     schema.TypeList,
+							Optional: true,
+							ForceNew: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+
+									"auto_repair": {
+										Type:     schema.TypeBool,
+										Optional: true,
+										ForceNew: true,
+									},
+
+									"auto_upgrade": {
+										Type:     schema.TypeBool,
+										Optional: true,
+										ForceNew: true,
+									},
+								},
+							},
+						},
+
+						"auto_scaling": {
```

Comment:
Can this be `autoscaling` to match what's in `google_container_node_pool` and the API docs (which has it as one word)?

Reply:
Modified as suggested